src/api/app/models/review.rb

```diff
@@ -98,38 +119,6 @@ def assigned_reviewer
     self[:reviewer] || by_user || by_group || by_project || by_package
   end
 
-  def check_initial
-    # Validates the existence of references.
-    # NOTE: they can disappear later and the review should be still
```

Comment:
isn't your change breaking exactly this NOTE? I.e. if we remove packages, we invalidate reviews ?

Reply:
Yes, @bgeuken you need to add `on: :create` to the`review_assignment` validation (as it was done in `check_initial` before) to keep the same behaviour. I agreee with @coolo that we may want to still display them.

Reply:
like 
```
irb(main):002:0> Review.where('by_package is not null and package_id is null').count
=> 3424

Reply:
Oh, that's the one thing I didn't check... what a mess

Reply:
well, we don't remove users and groups that often - otherwise you would see the same there